report/renderer/topbar-features.js

```diff
@@ -239,13 +239,17 @@ export class TopbarFeatures {
    * print dialog, all `<details>` are collapsed.
    */
   _setUpCollapseDetailsAfterPrinting() {
+    // We will feature detect w/ onbeforeprint, so gotta suppress typescript's narrowing.
+    /** @type {Omit<Window & typeof globalThis, 'onbeforeprint'> & {onbeforeprint?: any}} */
+    const self2 = self;
```

Comment:
nit: the naming isn't ideal here but I don't have a better idea. I'm fine leaving it if noone else can think of anything.